<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuranWidget</class>
 <widget class="QWidget" name="QuranWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Quran</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="cboChapter"/>
     </item>
     <item>
      <widget class="QLabel" name="chapterInfo">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spnVerse">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>3</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridSearch">
     <item row="0" column="1">
      <widget class="QPushButton" name="btnSearch">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="txtSearch"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="versesViewer"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
